README.md

```diff
@@ -39,6 +39,17 @@ To install our latest pre-release (3.0.0 alpha), run:
 ``` bash
 pip install -U --pre mesa
 ```
+With Mesa 3.0, we don't install all our dependencies anymore by default.
+```bash
+# You can customize the additional dependencies you need, if you want. Available are:
+pip install -U --pre mesa[network,viz]
+
+# This is equivalent to our recommended dependencies:
+pip install -U --pre mesa[rec]
```

Comment:
Hold on. This will trip people over. Visualization is what a lot of people coming to Mesa want to use it for. The default needs to include visualization. I can do `pip install solara` and have the common functionality working out of the box. Why do I have to do `pip install mesa[rec]` specifically for just Mesa?

Reply:
Doing a minimal install of Mesa is less common than doing an install of Mesa for visualization in a classroom setting. Can the split be `mesa-minimal`/`mesa-core` and `mesa` instead?

Reply:
I have multiple complex models that don't touch Solara, the batch_run or the DataCollector at all.

Also the initial error message when you can't install Solara is quite clear.

Also it's literally in our front-page Readme.

Reply:
You also typically want to avoid installing stuff like solara on an HPC. As long as it is well documented, which it is, I am fine with the current structure.

Reply:
I have similar situations (initially wanted to embed Mesa inside other simulation frameworks), but why make it more inconvenient to students and first time learners, who have to read the fine print of the README.md?

Reply:
We can put it in a few other places, like the visualization tutorial, and potentially also add a helpful warning on import somewhere.

Reply:
Sorry I also didn't saw this properly before, but I side with @rht on this one. Viz should be in core imho. Mesa should be "batteries included". For hpc or similar environments you can always install with "--no-deps". And in the future it might be more convinient to have a "mesa-core" package with minimal dependencies. But for now it should be included

Reply:
You know you where the person who opened https://github.com/projectmesa/mesa/issues/626 ;)

Let’s discuss at the next dev meeting.